mmv1/third_party/terraform/services/container/resource_container_cluster.go.erb

```diff
@@ -6147,6 +6145,21 @@ func containerClusterAutopilotCustomizeDiff(_ context.Context, d *schema.Resourc
 	return nil
 }
 
+// Default to VPC_NATIVE mode during initial creation
+func containerClusterNetworkingModeCustomizeDiff(_ context.Context, d *schema.ResourceDiff, meta interface{}) error {
+	// skip if not during initial creation
+	o, _ := d.GetChange("name")
```

Comment:
We checked name in BT instance, which I think was the precedent here. Are we able to check `id` instead? I'm not actually sure if it will work or not, but that field is generic across resources. Not a problem if resources have user-configured name fields like here and BT instance, but will save a headache if we inadvertently applied the same check to a resource where it's a post-create output.

Reply:
moved the logic to create, so no need for this check.